.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: ruby
 rvm:
-  - 1.8.7
```

Comment:
To avoid breaking compatibility and necessitating a major version bump, could you try explicitly setting the operating system distribution to Ubuntu 14.04 in the Travis CI configuration to match the previous matching build?

```yaml
dist: trusty
```

The [Travis CI list of precompiled Rubies](http://rubies.travis-ci.org/) seems to show that Rubies < 1.9.3 are unavailable on Ubuntu 16.